sdk/python/generative-ai/rag/notebooks/azure_cognitive_search/acs_mlindex_with_langchain.ipynb

```diff
@@ -773,7 +773,7 @@
     "data_source_glob = \"articles/machine-learning/**/*\"\n",
     "data_source_url = \"https://learn.microsoft.com/en-us/azure\"\n",
     "# This regex is used to remove the 'articles' folder from the source url and removes the file extension.\n",
-    "document_path_replacement_regex = r'{\"match_pattern\": \"(.*)/articles/(.*)(\\\\.[^.]+)$\", \"replacement_pattern\": \"\\\\1/\\\\2\"}'\n",
```

Comment:
Seems like the capture group for the file extension (which isn't used in the replacement) was deleted, on purpose?

Reply:
fair comment- lets go the otherwise